### ISSUE TITLE
Module with plus sign in name not listed

### ISSUE THREAD

**Issue by cmd-ntrf** [cmd-ntrf/jupyter-lmod#44]
Module like `rstudio-server/2021.09.1+372` are not listed by jupyter-lmod.

**Comment by cmd-ntrf** [comment]
The problem was the regex 
```
MODULE_REGEX_NO_HIDDEN = re.compile(r"^([^\W][\w\-_+.]*|[\w\-_+.\/]{1,}\/[\w][\w\-_.]*[^\/:])$", re.M)
```
that should have been
```
MODULE_REGEX_NO_HIDDEN = re.compile(r"^([^\W][\w\-_+.]*|[\w\-_+.\/]{1,}\/[\w][\w\-_+.]*[^\/:])$", re.M)
```

**Comment by cmd-ntrf** [comment]
Fixed by b12dd8a2f9ed918352b574ac8a6b46c7ba8dc987
